CHANGES.md

```diff
@@ -22,6 +22,10 @@
 
 ## Changes
 
+- Changes `JUPYTERHUB_VERSION` from `1.4.0-20210506` to `4.0.2-20230816`
```

Comment:
Add the note from your PR description as well: 
"jupyter lab google drive extension is not supported with this image"